.github/dependabot.yml

```diff
@@ -9,10 +9,11 @@ updates:
     schedule:
       interval: "weekly"
 
-  - package-ecosystem: "pip"
+  - package-ecosystem: "uv"
```

Comment:
While you're here, want to group all the dependencies so we get 1 PR per week rather than N?

https://github.blog/changelog/2023-06-30-grouped-version-updates-for-dependabot-public-beta/

Feel free to punt and I'll do this when I get to a computer.

Reply:
(let's group them all, not just Python)